src/colmap/util/file_test.cc

```diff
@@ -32,6 +32,7 @@
 #include "colmap/util/testing.h"
 
 #include <cstring>
+#include <thread>
```

Comment:
Why is this header needed?

Reply:
No, leftover from previous httplib implementation.